src/components/organisms/FAQItemBlock/index.tsx

```diff
@@ -20,8 +20,8 @@ const FAQItemBlock: React.FC<FAQItemBlockInterface> = (props: FAQItemBlockInterf
     return (
         <StyleFAQBlock>
             <FAQCategoryLabel content={props.section} />
-            {props.faqQuestions.map(item => (
-                <FAQItem question={item.question} answer={item.answer} itemID={item.itemID} key={item.itemID}/>
+            {props.faqQuestions.map((item) => (
+                <FAQItem question={item.question} answer={item.answer} itemID={item.itemID} key={`${item.itemID}` + Math.random()}/>
```

Comment:
Why using Math.random? `itemID` should be always unique.

Reply:
The console give me a 
Warning: Each child in a list should have a unique "key" prop
if I don't add the Math.random. I don't know why though, since like you say, it should be unique aleady.